packages/simm-helpers/src/convert-to-halfwidth.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable unicorn/numeric-separators-style */
```

Comment:
Filename phải trùng với tên export, ex: `to-halfwidth.ts`

Reply:
Bỏ nó bị lỗi này á a
![image](https://github.com/user-attachments/assets/07362477-5546-4e69-affc-7026b472157d)